src/summary_writer.jl

```diff
@@ -76,7 +99,30 @@ function Base.write(writer::FileWriter, graph::Graph)
     write(writer, event)
 end
 
+function set_default(writer::FileWriter)
+    context = tf.Context()
+    context.attrs["default_file_writer"] = writer
+    push!(tf.global_context, context)
+end
+
+function with_default(writer::FileWriter, block)
+    context = tf.Context()
+    context.attrs["default_file_writer"] = writer
+    tf.with_context(context, block)
```

Comment:
```suggestion
    tf.with_context(block, context)
```

to match above change for do-blocks